src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,93 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size;
+    private Node<K, V>[] table = (Node<K,V>[])new Node[DEFAULT_CAPACITY];
 
     @Override
-    public void put(K key, V value) {
 
+    public void put(K key, V value) {
+        int threshold = (int) (table.length * LOAD_FACTOR);
+        int hash = getHash(key);
+        int bucket = hash % table.length;
```

Comment:
you have the same logic in two places, let's create a separate method that will return number of bucket by Key and move this logic there